src/status_im/contexts/settings/keycard/style.cljs

```diff
@@ -1,22 +1,10 @@
 (ns status-im.contexts.settings.keycard.style
```

Comment:
This one has changed a lot. Could you please share the details?

Reply:
@ulisesmac, all the old styles in this file weren't used, by corresponding `view.cljs` got some inline styles. So I extracted them and put to `styles.cljs`